src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
-spring.graphql.graphiql.enabled=true
-
+spring.datasource.url=${DATASOURCE_URL:jdbc:mysql://localhost:3306/mmotodo}
+spring.datasource.username=${DATASOURCE_USERNAME:root}
+spring.datasource.password=${DATASOURCE_PASSWORD:root}
```

Comment:
Did not know you could do this to supply default environment variables. Very handy!

Reply:
Really neat to get it to work both in development/production and not needing different profiles. Or just as an option for some developers in the team to change the values without changing the properties file and making conflicts when merging commits.